src/Symfony/Component/Intl/Data/Generator/LanguageDataGenerator.php

```diff
@@ -84,6 +84,7 @@ class LanguageDataGenerator extends AbstractDataGenerator
         'zh' => 'zho',
     ];
     private static $blacklist = [
+        'root' => true,
```

Comment:
Could we add a very brief comment explaining what's this ... like we do for the rest of special "languages"? Thanks.